Sources/Build/BuildDelegate.swift

```diff
@@ -570,6 +571,14 @@ fileprivate struct CommandTaskTracker {
             totalCount -= 1
             break
         case .isComplete:
+            if (totalCount == finishedCount) {
+                let latestOutput: String? = latestFinishedText
+                latestFinishedText = """
+                \(latestOutput ?? "")\n
+                * Build Completed!
+                * Binaries: .build/ \n
```

Comment:
I'll take a look and see what it would take to get this information from the parameter that was used.  Because `swift build` has a `--build-path` parameter that can override this, the code can't assume that it will always be `.build/`.

Reply:
Thank you!  @abertelrud meanwhile I have added the failure message.

Reply:
Thanks, @federicobucchi.  It looks as if this source file would need to be restructured a little, or the printing of the message would need to be hoisted out, in order to properly support this.  So I think for this PR it might be better to leave out the build path, and maybe to file a JIRA at _bugs.swift.org_ about adding it as a follow-on.  Does that seem reasonable?  Adding the succeeded/failed would be a great improvement even without showing the build directory.

Reply:
https://bugs.swift.org/browse/SR-13764